eZ/Publish/Core/IO/IOMetadataHandler/LegacyDFSCluster.php

```diff
@@ -60,6 +56,11 @@ public function create(SPIBinaryFileCreateStruct $binaryFileCreateStruct)
     {
         $path = $this->addPrefix($binaryFileCreateStruct->id);
 
+        // Some code has been using integer, though DateTime is correct. Convert to DateTime if needed.
```

Comment:
shouldn't this rather be enforced if api doc says date time and so we can rather fix the use?